Mage/Archivist/Archivist.cs

```diff
@@ -463,7 +473,12 @@ public void MGet(JArray queries, JObject options, Action<Exception, JToken> cb)
 
 						// Add value to response
 						int responseKey = realQueryKeys[cacheKeyName];
-						responseArray[responseKey].Replace(GetCacheValue(cacheKeyName).Data);
+						var cacheValue = GetCacheValue(cacheKeyName);
+
+						if (cacheValue != null)
```

Comment:
This null check is no longer required as the delete will ensure the value exists in the cache
thus `cacheValue.Data` should return a null rather than a null exception
this check will swallow potentially valid errors